docs/instruments/ZJSONInstrument_v0.0.1.md

```diff
@@ -0,0 +1,970 @@
+described in more detail below.
+
+All keys in the Z-JSON Instrument follow the lowerCamelCase naming convention.
+inside each JSON object.
+
+## 2.1: Instrument Metadata
+
+```js
+{
+    "schema" : {
+        ...
+        "meta" : {
+            "createdBy": string,
+            "createdOn": string,
+            "createdWith": string,
+            "defaultLanguage": string,
+            "derivedFrom": string,
+            "importedFrom": string,
+            "includeMetaDataFields": boolean,
+            "instrumentVersion": string,
+            "lastUpdateOn": string,
+            "longName": {
+                "languageTag1": string,
+                "languageTag2": string,
+                ...
+            },
+            "previousVersion": string,
+            "schemaVersion": string,
+            "shortName": string
+        },
+        ...
+    },
+    ...
+}
+```
+
+The metadata object contains information such as the version of the instrument, its name, its description, and
+some provenance data. The value of each key is defined as such:
+
+`meta.createdBy`: String.
+
+The developer or user responsible for creating the Z-JSON Instrument.
+
+`meta.createdOn`: String.
+
+The date of creation of the Z-JSON Instrument in the format 'YYYY-MM-DD'. Each new version of an instrument will have a new `createdOn` date. Small
+changes, such as bug fixes, that do not require the creation of a new version of an instrument should use `meta.lastUpdateOn`
+instead to track updates.
+                 
+`meta.createdWith`: String.
+
+The software or tool responsible for the creation of the ZJSON Instrument i.e. the Instrument Builder module or some conversion script.
+
+`meta.defaultLanguage`: Required String.
+                        
+The default (human communication) language of the instrument, represented by an IETF BCP 47 (RFC 5646 Standard) language tag e.g. "en-CA".
+It is important to set the default language as it is used as a key in other locations of the `schema` object.
+
+`meta.derivedFrom`: String.
+                    
+The resource instrument that the Z-JSON Instrument derived from as a result of significant content modification such that the instrument is no longer the same                    
+instrument, and the creation of a new version does not suffice. What is considered significant here is not defined in this specification, but is instead up to the author to decide.
+                    
+This level of modification does not include format transcription i.e. linst to
+JSON, unless the content of the instrument has also been modified. The
+`meta.importedFrom` key should be used in the case of conversions.
+
+`meta.importedFrom`: String.
+
+The resource instrument that this Z-JSON Instrument was converted from, if it was imported from another format and is otherwise unmodified. If any modifications to the content of the instrument are madeafter conversion, this value should be empty and `meta.derivedFrom` used instead.
```

Comment:
```suggestion
The resource instrument that this Z-JSON Instrument was converted from, if it was imported from another format and is otherwise unmodified. If any modifications to the content of the instrument are made after conversion, this value should be empty and `meta.derivedFrom` used instead.
```